config.json

```diff
@@ -324,6 +324,10 @@
     "pdp-non-sale-price-label": "",
     "pdp-retail-price-label": "",
     "pdp-custom-fields-tab-label": "",
+    "paymentbuttons-paypal-accelerated-checkout-color": "black",
+    "paymentbuttons-paypal-accelerated-content-color": "white",
+    "paymentbuttons-paypal-accelerated-content-label": "checkout",
+    "paymentbuttons-paypal-accelerated-border-color": "black",
```

Comment:
It might be better to set an empty value here, because some stores can have buttons without border

Reply:
This was indicated in the [task](https://bigcommercecloud.atlassian.net/browse/PAYPAL-1653).@nkostohryz What do you think about it?

Reply:
Well, black border works perfectly with black button color. I think it would be beneficial for merchants to have all these settings combination used by default, inc. the border color. After all, it is a bad UX to have settings unset by default.

Reply:
Well, black border works perfectly with black button color. I think it would be beneficial for merchants to have all these settings combination used by default, inc. the border color. After all, it is a bad UX to have settings unset by default.